src/EditorFeatures/CSharpTest/EditAndContinue/TopLevelEditingTests.cs

```diff
@@ -3448,6 +3448,68 @@ public void Method_ReadOnlyRef_ReturnType_Update()
                 Diagnostic(RudeEditKind.TypeUpdate, "ref readonly int M()", FeaturesResources.method));
         }
 
+        [Fact]
+        public void Method_ImplementingInterface_Add()
```

Comment:
>public void Method_ImplementingInterface_Add( [](start = 8, length = 45)

Can you also add a test that validates that adding a new class that implements an interface explicitly does not report any rude edits?

Reply:
Thank you! Added

---
In reply to: [213479396](https://github.com/dotnet/roslyn/pull/29569#discussion_r213479396) [](ancestors = 213479396)